src/fastoad_cs25/models/weight/mass_breakdown/cs25.py

```diff
@@ -47,6 +50,8 @@ def setup(self):
         self.add_input("data:load_case:lc2:U_gust", val=np.nan, units="m/s")
         self.add_input("data:load_case:lc2:altitude", val=np.nan, units="ft")
         self.add_input("data:load_case:lc2:Vc_EAS", val=np.nan, units="m/s")
+        self.add_input("data:load_case:manoeuvre_load_factor", val=2.5)
+        self.add_input("data:load_case:GLA_intensity", val=1.0)
```

Comment:
I wonder, should they be in "data:*" ?
I would rather use `"settings:load_case:manoeuvre_load_factor"`.
For GLA, maybe "tuning:*" is better. But more important, the name here is confusing wrt current computation. For alleviation intensity = 1, we would expect gust load factor to be 0.

Reply:
As discussed, the new variables remain in "data:load_cases" alongside the old ones and GLA_intensity was renamed to gust_intensity. Variable description added in [d0f3099](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/38/commits/d0f3099109c8fb3f4c9d1e683b98a6755a0e8e9d)